slave-new/process-o365/conf/pre_01_setup_access_token

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Rename this file to example-pre_01_setup_access_token. We want the system administrator to create pre scripts on their own. Also don't want to rewrite any changes in pre scripts with update of DEB/RPM package.